templates/pharmacies/general_edit.html

```diff
@@ -1,73 +1,78 @@
-<table style="font-size:9pt;" width="500px" CELLSPACING="0" CELLPADDING="3">
-<tr>
-	<td width="150px" VALIGN="MIDDLE" >{xl t='Name'|escape:'html'}</td>
-	<td VALIGN="MIDDLE" >
-		<input type="text" size="40" name="name" value="{$pharmacy->name|escape:'html'}" onKeyDown="PreventIt(event)" />({xl t='Required'|escape:'html'})
-	</td>
-</tr>
-<tr>
-	<td VALIGN="MIDDLE" >{xl t='Address'|escape:'html'}</td>
-	<td VALIGN="MIDDLE" >
-		<input type="text" size="40" name="address_line1" value="{$pharmacy->address->line1|escape:'html'}" onKeyDown="PreventIt(event)" />
-	</td>
-</tr>
-<tr>
-	<td VALIGN="MIDDLE" >{xl t='Address'|escape:'html'}</td>
-	<td VALIGN="MIDDLE" >
-		<input type="text" size="40" name="address_line2" value="{$pharmacy->address->line2|escape:'html'}" onKeyDown="PreventIt(event)" />
-	</td>
-</tr>
-<tr>
-	<td VALIGN="MIDDLE" >{xl t='City, State Zip'|escape:'html'}</td>
-	<td VALIGN="MIDDLE" >
-		<input type="text" size="25" name="city" value="{$pharmacy->address->city|escape:'html'}" onKeyDown="PreventIt(event)" /> , <input type="text" size="2" maxlength="2" name="state" value="{$pharmacy->address->state|escape:'html'}" onKeyDown="PreventIt(event)" /> <input type="text" size="5" name="zip" value="{$pharmacy->address->zip|escape:'html'}" onKeyDown="PreventIt(event)" />
-	</td>
-</tr>
-<tr>
-	<td VALIGN="MIDDLE" >{xl t='Email'|escape:'html'}</td>
-	<td VALIGN="MIDDLE" >
-		<input TYPE="TEXT" NAME="email" SIZE="35" VALUE="{$pharmacy->email|escape:'html'}" onKeyDown="PreventIt(event)" />
-	</td>
-</tr>
-<tr>
-	<td VALIGN="MIDDLE" >{xl t='Phone'|escape:'html'}</td>
-	<td VALIGN="MIDDLE" >
-		<input TYPE="TEXT" NAME="phone" SIZE="12" VALUE="{$pharmacy->get_phone()|escape:'html'}" onKeyDown="PreventIt(event)" />
-	</td>
-</tr>
-<tr>
-	<td VALIGN="MIDDLE" >{xl t='Fax'|escape:'html'}</td>
-	<td VALIGN="MIDDLE" >
-		<input TYPE="TEXT" NAME="fax" SIZE="12" VALUE="{$pharmacy->get_fax()|escape:'html'}" onKeyDown="PreventIt(event)" />
-	</td>
-</tr>
-
-<tr>
-	<td VALIGN="MIDDLE" >{xl t='Default Method'|escape:'html'}</td>
-	<td VALIGN="MIDDLE" >
-		<select name="transmit_method">{html_options options=$pharmacy->transmit_method_array selected=$pharmacy->transmit_method}</select>
-	</td>
-</tr>
-<tr><td colspan="2">&nbsp;</td></tr>
-<tr>
-	<td colspan="2">
-	    <div class="btn-group">
-	        <a href="javascript:submit_pharmacy();" class="btn btn-default btn-save" onclick="top.restoreSession()">
-	            {xl t='Save'|escape:'html'}
-	        </a>
-	        <a href="controller.php?practice_settings&pharmacy&action=list" class="btn btn-link btn-cancel" onclick="top.restoreSession()">
-                {xl t='Cancel'|escape:'html'}
-            </a>
-        </div>
-	</td>
-</tr>
-</table>
-<input type="hidden" name="id" value="{$pharmacy->id|escape:'html'}" />
-<input type="hidden" name="process" value="{$PROCESS|escape:'html'}" />
+    <input type="hidden" name="form_id" value="{$pharmacy->id|escape:'html'}" />
+	<div class="form-group">
```

Comment:
From the HTML spec:

> Tables should not be used as layout aids. Historically, many Web authors have tables in HTML as a way to control their page layout making it difficult to extract tabular data from such documents. In particular, users of accessibility tools, like screen readers, are likely to find it very difficult to navigate pages with tables used for layout. If a table is to be used for layout it must be marked with the attribute role="presentation" for a user agent to properly represent the table to an assistive technology and to properly convey the intent of the author to tools that wish to extract tabular data from the document. There are a variety of alternatives to using HTML tables for layout, primarily using CSS positioning and the CSS table model. [CSS]
https://www.w3.org/TR/html5/tabular-data.html#tabular-data

From only the above, the burden of defending the layout should fall squarely on the use of `table`, not HTML/CSS. Is there a time where the use of a table should be used to layout non-tabular data? Maybe, but it's the exception to the rule.

As I've stated before, table-based layouts have not been acceptable practice for near a decade, if not longer. They hinder modern development. The standing rule should be "Use `divs` and CSS unless using tabular data."

This should not decided on a case-by-case basis. We should use modern development standard.

We've for far too long been lax about standards, both in _how_ we write code (read: code styling) and _what_ we write (read: table-based layouts). Finally getting PSR in place has been a huge stride for the project, and dropping these tables will be another huge step.